### DISCUSSION TITLE
Support for Inpaint and Regional Prompt

### DISCUSSION THREAD

**Discussion by sadume** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#217] [General]
Inapaint doesn't seem to work with regional prompt is this correct? Maybe im doing something wrong. Or an ideas if this will be supported in the future.

Thanks.

**Comment by sebaxakerhtc** [comment]
It works fine for me.
You should do all the same settings like for img2img. Then draw mask and go!
